src/org/parosproxy/paros/core/scanner/VariantJSONQuery.java

```diff
@@ -294,6 +288,23 @@ private void parseToken(String token) {
         }
     }
 
+    private void readEscapedString(){
+        int chr;
+        while (true) {
+            chr = sr.read();
+            if (chr == BACKSLASH) {
+                sr.read();
+                continue;

```

Comment:
This statement could be removed (does not add much, it already goes to next cycle).